public/index.js

```diff
@@ -21,11 +21,11 @@ Vue.createApp({
 		},
 		submit: function () {
 			if (!this.idCode) return;
+			this.showInstructions = false;
```

Comment:
This behavior is a lot more confusing imho because the spinner is quite small and the instructions are replaced with "not found" error even though it's still loading.

Reply:
Moved the instructions flag to the .finally()